### PR TITLE
Adds portable seed generator to ashwalkers

### PR DESCRIPTION
**Pull request by Daturix**
title

### CODE REVIEW

File: _maps/RandomRuins/LavaRuins/lavaland_surface_ash_walker1.dmm [hunk 1]
```diff
@@ -1,588 +1,65 @@
-//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```
Comment (Quardbreak) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_ash_walker1.dmm]: 👍

Comment (Quardbreak) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_ash_walker1.dmm]: seems you dont use mapmerge, or i something dont notice

Comment (Daturix) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_ash_walker1.dmm]: didnt use mapmerge

**Comment by Haevacht** [comment]
ALso, >merging from master.

**Pull request closed without merging**